en/docs/learn/consume-api/account-management-dev-portal/change-dev-portal-password.md

```diff
@@ -0,0 +1,21 @@
+# Change Password for the Developer Portal
```

Comment:
Do we really need to document this? it's quite intuitive I think as you mentioned offline.

Reply:
Decided to keep it based on an offline discussion with Nuwan